src/main/python/pgshovel/replication/streams/kafka.py

```diff
@@ -29,7 +36,12 @@ def consume(self, state):
         # hard we don't really need to deal with it right now.
         assert len(consumer.offsets) is 1
 
-        if state.HasField('stream_state'):
+        decoded = imap(
+            lambda (offset, msg): tuple((offset, self.codec.decode(msg.value))),
```

Comment:
Hmmm I had some weirdness with using the implicit `(,)` syntax before where I got a `SyntaxError` but I can't remember what that was.  Will fix.